runtime/tests/utils/utils.go

```diff
@@ -192,3 +192,6 @@ func AssertValueSlicesEqual(t testing.TB, inter *interpreter.Interpreter, expect
 
 	return true
 }
+
+// Noop statement for placing a breakpoint during debugging.
+func Breakpoint() {}
```

Comment:
Is this change intended?

Reply:
ah yea, just a useful oneliner that wouldn't make for much of a PR on its own. i can remove it

Reply:
Is this for go-debugging?
Given that is not used anywhere - and sort of not related to the objective of the PR, would be better to remove it from the PR.